src/cudamatrix/cu-matrix.cc

```diff
@@ -26,6 +26,7 @@
 
 
 #if HAVE_CUDA == 1
+#include <nvToolsExt.h>
```

Comment:
is this needed?

Reply:
looks like this is left over from profiling.  it can be safely removed.